File: Iniciante/1146 - Sequências Crescentes/JavaScript/1146.js
function main() {
    const input = require('fs').readFileSync(0, 'utf-8').trim().split('\n');

    let index = 0;

    while (true) {
        let x = Number(input[index++]);

        if (x === 0) {
            return;
        }

        let result = [];

        for (let i = 1; i <= x; i++) {
            result.push(i);
        }

        console.log(result.join(' '));
    }
}

main();

File: Iniciante/1146 - Sequências Crescentes/Python/1146.py
def main():
    while True:
        x = int(input())

        if x == 0:
            return

        result = []

        for i in range(1, x + 1):
            result.append(str(i))

        print(" ".join(result))

if __name__ == "__main__":
    main()